packages/ref-metrics-pmp/src/cmip_ref_metrics_pmp/variability_modes.py

```diff
@@ -105,25 +105,47 @@ def build_cmd(self, definition: MetricExecutionDefinition) -> Iterable[str]:
         logger.debug(f"reference_dataset_name: {reference_dataset_name}")
         logger.debug(f"reference_dataset_path: {reference_dataset_path}")
 
+        model_files = input_datasets.path.to_list()
+
+        if len(model_files) != 1:
+            # Have some logic to replace the dates in the filename with a wildcard
+            raise NotImplementedError("Only one model file is supported at this time.")
+
+        if isinstance(model_files, list):
+            modpath = " ".join([str(p) for p in model_files])
+        else:
+            modpath = model_files
+
+        if isinstance(reference_dataset_path, list):
+            reference_data_path = " ".join([str(p) for p in reference_dataset_path])
+        else:
+            reference_data_path = reference_dataset_path
+
+        # Build the command to run the PMP driver script
         params = {
             "driver_file": "variability_mode/variability_modes_driver.py",
             "parameter_file": self.parameter_file,
-            "model_files": input_datasets.path.to_list(),
-            "reference_name": reference_dataset_name,
-            "reference_paths": reference_dataset_path,
-            "source_id": source_id,
-            "experiment_id": experiment_id,
-            "member_id": member_id,
-            "output_directory_path": str(definition.output_directory),
             "variability_mode": self.mode_id,
+            "modpath": modpath,
+            "modpath_lf": "none",
+            "exp": experiment_id,
+            "realization": member_id,
+            "modnames": source_id,
+            "reference_data_name": reference_dataset_name,
+            "reference_data_path": reference_data_path,
+            "results_dir": str(definition.output_directory),
+            "cmec": None,
+            "no_provenance": None,
         }
 
         # Add conditional parameters
         if self.mode_id in ["SAM"]:  # pragma: no cover
             params["osyear"] = 1950
             params["oeyear"] = 2005
 
-        development_mode = False
+        print("jwlee123_test-1 params:", params)
```

Comment:
I would recommend using the debugger to run the test and add breakpoints wherever you need. That way you don't end up with these extra print statements

Reply:
Resolved by the following commit: https://github.com/Climate-REF/climate-ref/pull/221/commits/779677483bf344ceec21a691bb3120b077bb3864